performance/pom.xml

```diff
@@ -21,6 +21,9 @@
     <description>Hibernate Validator performance tests.</description>
 
     <properties>
+        <!-- Disable enforcer as we are using banned dependencies for old versions -->
+        <enforcer.skip>true</enforcer.skip>
```

Comment:
Did you mean to commit this? Not a problem but wondering, as I see no direct relation.

Reply:
Yes. Otherwise the benchmarks can't be run as we have banned the old dependencies used by the benchmarks from HV (the javax ones) but we still need them for the old versions benchmarks. It's in a specific commit tweaking the benchmarks.